build/scripts/build-utils.ps1

```diff
@@ -56,7 +105,7 @@ function Ensure-BasicTool([string]$name, [string]$version) {
     $p = Join-Path (Get-PackagesDir) "$($name).$($version)"
     if (-not (Test-Path $p)) {
         $nuget = Ensure-NuGet
-        Exec { & $nuget install $name -OutputDirectory (Get-PackagesDir) -Version $version }
+        Exec-Block { & $nuget install $name -OutputDirectory (Get-PackagesDir) -Version $version } | Out-Null
```

Comment:
Just depends on whether the output should / shouldn't be placed onto the screen.  Most of these function are called eventually from user invoked scripts: Restore.cmd, Build.cmd, etc ...  The use of `Out-Null` vs. `Out-Host` is mostly around ensuring that experience is reasonable.